grafana_objects/client_grafana_dashboard_structs.go

```diff
@@ -0,0 +1,66 @@
+package grafana_objects
+
+type DashboardMeta struct {
+	IsStarred bool   `json:"isStarred"`
+	Url       string `json:"url"`
+	FolderId  int    `json:"folderId"`
+	FolderUid string `json:"folderUid"`
+	Slug      string `json:"slug"`
+}
+
+type DashboardObject struct {
+	Id            int            `json:"id"`
+	Uid           string         `json:"uid"`
+	Title         string         `json:"title`
+	Tags          []string       `json:"tags"`
+	Style         string         `json:"style"`
```

Comment:
Some of those fields do not appear in the [documentation](https://docs.logz.io/api/#operation/createDashboard) (Style, Editable, GraphToolTip, Time, Timepicker, Templating, Annotations, Panels). Why did you add those?

Reply:
Hello @mirii1994, I found those in https://grafana.com/docs/grafana/latest/dashboards/json-model/